addons/medical_damage/initSettings.sqf

```diff
@@ -34,6 +34,15 @@
     true
 ] call CBA_fnc_addSetting;
 
+[
+    QEGVAR(medical,painUnconsciousThreshold),
```

Comment:
```suggestion
    QUOTE(PAIN_UNCONSCIOUS),
```
technically it could be this I don't really like it and would rather keep it as is